contracts/UFragmentsPolicy.sol

```diff
@@ -67,6 +67,14 @@ contract UFragmentsPolicy is Ownable {
     // The number of rebase cycles since inception
     uint256 public epoch;
 
+    // The beginning of rebase time window within minRebaseTimeIntervalSec cyclic period.
+    // If minRebaseTimeIntervalSec = 24 hours, it's time of the day from midnight in seconds,
+    // at which rebase window begins.
+    uint256 public rebaseWindowOffsetSec;
```

Comment:
Can we move these two new rebase properties up above epoch, so they're with the other two?

Reply:
👍